temp/legacy-code/src/main/java/com/ivy/legacy/legacy/ui/theme/modal/edit/AmountModal.kt

```diff
@@ -76,10 +76,10 @@ fun BoxWithConstraintsScope.AmountModal(
         mutableStateOf(
             if (currency.isNotEmpty()) {
                 initialAmount?.takeIf { it != 0.0 }?.format(currency)
-                    ?: ""
+                    ?: "0"
             } else {
                 initialAmount?.takeIf { it != 0.0 }?.format(decimalCountMax)
-                    ?: ""
+                    ?: "0"
```

Comment:
That's not desired, let's keep it empty so it works and the user sees the hint. Why do we want "0"?

Reply:
oh my bad, i already remove it in previous PR. Maybe when I reverted to capture the old UI, I accidentally added it again

Reply:
@isaacnguyen0809 can you retest that the Amount modal and calculator works well and looks good in:
- empty input-> open calc -> type
- enter amount -> open calc -> type
- cases like this

A screen recording will be appreciated and then we'll merge.

Reply:
here is the result :
https://github.com/Ivy-Apps/ivy-wallet/assets/57831153/14587150-7707-49c7-935d-d283b1b91c5c

Reply:
Merging 💯 Just to check quickly it work when you click "Set" right?

Reply:
https://github.com/Ivy-Apps/ivy-wallet/assets/57831153/25260ae0-6c24-41ee-8dc0-e1c0f9046245

yes, it worked perfectly